packages/client/hmi-client/src/composables/useClientEvent.ts

```diff
@@ -14,3 +16,13 @@ export function useClientEvent(
 		await Promise.all(eventTypes.map((type) => unsubscribe(type, handlerFn)));
 	});
 }
+
+// accepts a string[] to update with in progress task ids
+export function createClientEventHandler(taskIds: string[]) {
```

Comment:
what is the advantage to pass a list `taskIds` here instead of a single `taskId`?